### DISCUSSION TITLE
Open file slow in VDI

### DISCUSSION THREAD

**Discussion by rickywu** [ayamir/nvimdots#1050] [Q&A]
Nvim works well on my PC, but works slow on VDI which is a cloud desktop env.

such as accelerate j/k move slow, open files need 1-2 seconds, I can't bear.
Any ways to find out the reason? I tried increas CPU cores and memory, but no change.

**Comment by Jint-lzxy** [comment]
Do you have a decent internet connection?

**Comment by rickywu** [comment]
No, startup without any plugin, works well

**Reply by rickywu** [reply]
I found the reason is some client security app inject nvim process, now the only problem is press keyboard got latency about 10ms, seems  caused by network.
